### PR TITLE
Unicode support

### PR DESCRIPTION
**Pull request by mkassner**

### CODE REVIEW

**Comment by mkassner** [comment]
@papr do you see a reason not to merge?

File: fontstash.pyx [hunk 1]
```diff
@@ -12,6 +12,18 @@ FONS_ALIGN_BASELINE = fs.FONS_ALIGN_BASELINE
...
+        return <unicode>s
+    else:
+        return (<bytes>s).decode('utf-8')
+
+cdef bytes _utf8_string(basestring s):
+    if type(s) is unicode:
+        return (<unicode>s).encode('utf-8')
+    else:
+        return <bytes>s
+
```
Comment (papr) [inline fontstash.pyx:26]: How about calling these `_encoded_string` and `_decoded_string`?

Comment (mkassner) [reply fontstash.pyx:26]: I think its important to mention the code (utf-8 )

Comment (papr) [reply fontstash.pyx:26]: Ok, I see that. But the difference is not clear in `ustring` vs `utf8_string`.  How about `_utf8_enc_str` and `_utf8_dec_str`?

Comment (mkassner) [reply fontstash.pyx:26]: how about:

_to_unicode 
_to_utf8_bytes

because technically unicode is not a char array.

Comment (papr) [reply fontstash.pyx:26]: We can agree on that.

**Pull request merged**